core/handler/call_hierarchy.py

```diff
@@ -0,0 +1,67 @@
+from core.handler import Handler
+from core.utils import *
+
+class CallHierarchy(Handler):
+    name = "call_hierarchy"
+    method = "textDocument/prepareCallHierarchy"
+
+    income = True
+
+    wait_response = 0
+    call_contents = []
+
+    def reset_handler(self):
+        self.send_document_uri = True
+        self.method = "textDocument/prepareCallHierarchy"
+        self.income = True
+        self.call_contents = []
+
+    def process_request(self, position_or_items, income=True) -> dict:
+        if self.method == 'textDocument/prepareCallHierarchy':
+            self.income = income
+            return dict(position=position_or_items)
+        else:
+            return dict(item=position_or_items)
+
+
+    def add_call_content(self, item, from_ranges):
+        self.call_contents.append({'name': item['name'],
+                                    'path': uri_to_path(item['uri']),
+                                    'icon': SYMBOL_MAP[item['kind']].lower(),
```

Comment:
We should use KIND_MAP here, we should add key in KIND_MAP if some key miss here (such as, Namespace etc)

When some missed key add in KIND_MAP, we should add those key in `acm-icon-alist` and call command `acm-icon-fetch-all` to fetch icon of missed key.

Reply:
Call hierarchy and completion use different kind map according document.
<img width="644" alt="image" src="https://user-images.githubusercontent.com/19282154/202886415-6a4190da-36af-4287-a5c9-93ab2ebe31be.png">
<img width="645" alt="image" src="https://user-images.githubusercontent.com/19282154/202886421-0b7bda1f-2ebf-420e-935e-923a4e67d57f.png">

Reply:
I see.

Reply:
You just need  put some missed key add in acm-icon-alist and call command acm-icon-fetch-all to fetch icon of missed key.

Reply:
ok